### DISCUSSION TITLE
Demonstrate code for including without actually including

### DISCUSSION THREAD

**Discussion by wwarriner** [mondeja/mkdocs-include-markdown-plugin#127] [Q&A] [answered]
Great plugin, this is helpful!

Our community documentation has a contributor guide. In that guide I want to show others how to use this plugin as it applies to our documentation. To do so I've included this:

````
  ```text
  {%
      include-markdown "../docs_include/<file>.md"
      heading-offset=<parent-section-header-level>
  %}
  ```
````

However, it (naturally) gets interpreted as an inclusion, which creates an error and an empty fenced code block in the rendered page.

Is there functionality to instruct `mkdocs-include-markdown-plugin` to ignore this particular inclusion so that I can render the contents of the fenced code block verbatim?

**Comment by mondeja** [comment, marked as answer]
You can include the code block itself from other files. The plugin should skip code blocks looking for nested inclusions so:

- *includer.md*
   ```
   {% include-markdown "./included.md" %}
   ```
- *included.md*
   ````
   ```
   {%
      include-markdown "../docs_include/<file>.md"
      heading-offset=<parent-section-header-level>
   %}
   ```
   ````

**Reply by iain-palmer** [reply]
Hi, I have a similar use-case. I've tried the above answer, but I get the following error:
```
ERROR   -  Error reading page 'included.md':
ERROR   -  No files found including '../docs_include/<file>.md' at included.md:2
```
Any suggestions?

**Reply by mondeja** [reply]
It would be a bug. Fill an issue sharing your code.

**Reply by DeNelo** [reply]
Isn't that because recursive=true?

**Reply by DeNelo** [reply]
Sorry, answering an old post!

**Comment by DeNelo** [comment]
In such situations, I add a [zero-width space character](https://unicode-explorer.com/c/200Bd) between `{` and `%`. That keeps mkdocs from interpreting what you write.
